src/facade/dragonfly_connection.h

```diff
@@ -116,6 +116,11 @@ class Connection : public util::Connection {
     util::fb2::BlockingCounter bc;  // Decremented counter when processed
   };
 
+  struct InvalidationMessage {
+    std::string_view key;
```

Comment:
I believe you should use a string here and not string_view.
Who is the owner of this string and how do you know it is alive until you send the message?

Reply:
fixed.